File: PY421/code/celsius2.c
#include <stdio.h>

/* Print a Fahrenheit-Celsius conversion table */

main()
{
  int fahr;
  float celsius;
  int lower=0, upper=300, step=20;

  fahr=lower;
  while(fahr<=upper){
    celsius=5.0*(fahr-32)/9.0;
    printf("%4d%8.2f\n", fahr, celsius);
    fahr=fahr+step;
  }
}

File: PY421/code/celsius2.f90
PROGRAM celsius2

! Print a Fahrenheit-Celsius conversion table 

  IMPLICIT NONE  

  IntEgER fahr
  REAL celsius
  INTEGER :: lower=0, upper=300, step=20
  
  fahr=lower
  DO WHILE(fahr<=upper)
     celsius=5.0*(fahr-32)/9;   !semicolon not needed
     PRINT '(I4,F8.2)', fahr, celsius
     fahr=fahr+step
     
  END DO
  
END PROGRAM celsius2

